### ISSUE TITLE
Windows 10 Access Denied

### ISSUE THREAD

**Issue by trivii** [basildane/WakeOnLAN#38]
I'm trying to get a WOL system running on 8 PC's, All are Windows 10. I've gone through the configuration steps of enabling the magic packet on the network adapters, but every time I try to shut down a PC, I always get the same error. 

"Error: Access is denied. (Exception from result HRESULT: 0x80070005(E_ACCESSDENIED))"

Could anyone point me in the right direction? Does the software need to be installed on each client?

**Comment by basildane** [comment]
You do not need any software on the clients.  Your account must have authority to shutdown the remote computers.  Please start here.  [Help - Shutdown](http://wol.aquilatech.com/help/shutdown/default.html)

It is a little easier if the computers are in a domain.  Are they?  Are they on the same network subnet?

**Comment by trivii** [comment]
So they are not on a network domain. Quite frankly, I'm a complete novice here. I don't believe they are located in a workgroup either. They do all however have a designated IP Adress with the same default gateway, and subnet mask. Should I be changing all of the usernames and passwords to the same exact one throughout each machine?

**Comment by trivii** [comment]
I added the registery key(was missing - set hexidecimal to 1), changed bios settings, enabled network adapter. Changed both this PC, and PC I wish to control to the same username/ no password, am running WOL software on this pc as right click, run as admin.

**Comment by basildane** [comment]
Let's backup and simplify things.
The computer you are running WOL on, that user account must have the rights to shutdown the remote computers.  This is done in the operating system.  In a domain it is done with group policy.  If you use an account that is an admin account, that probably has the rights to shutdown the computers.

Some systems that are in a workgroup only seem to work using "legacy" shutdown method.  This is set in the "Shutdown" tab in the properties for the remote machine.

Let's take WOL completely out of the equation for testing.
From a command prompt, enter `shutdown /i` and shutdown the remote computer using that program.  That uses the legacy shutdown method.  If it doesn't work, you need to fix that first.

**Comment by trivii** [comment]
I think I did it correctly, it returned that access is denied. 
![wol](https://cloud.githubusercontent.com/assets/23175134/19901230/cc0b0e78-a03c-11e6-8548-d454e42b5e5f.png)


Have to say I appreciate your time and patience with me in this.

**Comment by trivii** [comment]
What is interesting is that I can wake up the computer from sleep. Can not wake up from shut down, can not shut down, or put to sleep though.

**Comment by basildane** [comment]
The wake up from shutdown is another problem entirely, probably need to update your NIC drivers and enable wake from magic packet.  

Before you can get shutdown to work, you have to have a windows user account on your local computer that has authority to shutdown the remote computer.  Once it works with the windows command it should be able to work with other software like WOL.   You cannot shutdown a remote computer without authorization, obviously that would be big security problem if you could.

**Comment by Moopere** [comment]
> What is interesting is that I can wake up the computer from sleep. Can not wake up from shut down, can not shut down, or put to sleep though.

Yes, I've seen this a million times.  Are these older machines upgraded to Win10?  By older I mean more than 12 months old?  If so, chances are high that the Win8/8.1 and 10 drivers simply won't wake the machine from an off state.  I first had this hit me with older Dells during win8.0's time.

The only solution I've found that consistently works is to downgrade your NIC drivers.  I'll start with the last Win8.1 driver and if that doesn't work, go to the last win8 driver ... still not working?  Go to the Win7 version.  The Win7 driver will -always- work,

You will need to watch these machines closely going into the future as if you allow Win10 to automatically upgrade drivers (as is its default) it will again stop your WOL working.

I've forcibly turned off all Windows 10 updates on all machines I administer because its otherwise impossible to keep control of them.

**Comment by dayneh** [comment]
I am having the same problem I can transfer files form all three computers. I can remote desktop to my server and my recorder but I get the same error when I try to Hibernate them.
I have the same account on all 3 computers as admin.
I am in a workgroup called HOME1. 2 computers are running Win 10Pro 64 bit 1 is Win 10 Pro 32 bit.
I tried shutdown /i from a administrator command prompt the first time tried I got the a ACCESSDENIED
so I dided the regedit thing now I get "server: Failed to connect to registry."
